connectivity/drivers/nfc/TARGET_M24SR/source/m24sr_driver.cpp

```diff
@@ -0,0 +1,1464 @@
+/**
+ ******************************************************************************
+ * @file    m24sr_driver.cpp
+ * @author  ST Central Labs
+ * @brief   This file provides a set of functions to interface with the M24SR
+ *          device.
+ ******************************************************************************
+ * @attention
+ *
+ * <h2><center>&copy; COPYRIGHT(c) 2018 STMicroelectronics</center></h2>
```

Comment:
Needs SPDX (probably added in a separate commit, so we can still compare the deletion in the example repo, with the addition in mbed-os, as being identical)